src/ProductsXmlFeed.php

```diff
@@ -79,11 +81,32 @@ public static function get_xml_item( $product ) {
 			}
 		}
 
+		$xml .= self::get_attributes_xml( $product, "\t\t\t" );
```

Comment:
Could we call it like any other field through the loop over `self::$feed_item_structure` ? Not breaking this pattern would be the best option imo.

Reply:
That makes sense. My original thought was to just let the attribute manager class to handle all attributes (in this PR we only add one but there are more to be added).

But if we prefer to stick with the structure then we can add a function here for each attribute we add.